ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/InteractiveStatementExecutorTest.java

```diff
@@ -730,9 +699,6 @@ public void shouldRestoreLegacyRunScriptCommand() {
             0L
         )
     );
-
-    // Then:
-    verify(mockParser, mockEngine, mockQuery);
```

Comment:
Can we verify the `mockEngine.execute()` call expected in `mockReplayRunScript()`? That seems to be what this test was meant to check.

Reply:
I added a verify that the query isn't started. I think that's what should be tested since the only difference between `shouldRestoreLegacyRunScriptCommand` and `shouldHandleLegacyRunScriptCommand` is the mode, which causes `handleLegacyRunScript` in the executor to act differently.